ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -391,19 +391,21 @@ objects:
         #!/bin/bash
         set -euo pipefail
 
-        trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/shared/setup-success; else touch /tmp/shared/exit /tmp/shared/setup-failed; fi; exit "${rc}"' EXIT
+        trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/setup-success; else touch /tmp/exit /tmp/setup-failed; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
-        if [[ "${CLUSTER_TYPE}" != "gcp" ]] && [[ "${CLUSTER_ENABLE_NESTED_VIRT-false}" == "true" ]]; then
-            echo "CLUSTER_ENABLE_NESTED_VIRT is unsupported for cluster type '${CLUSTER_TYPE}'"
-            exit 1
+        if [[ "${CLUSTER_ENABLE_NESTED_VIRT}" ]]; then
```

Comment:
Why did you have to change it back to not using the `${-}` syntax? I guess this will not work with using the `set -u`? I'd prefer not to use nested ifs to reduce complexity for the reader.